client/src/pages/NeedHelp.js

```diff
@@ -111,7 +111,8 @@ const Step3 = (props) => {
       <StepTitle>What is your email address?</StepTitle>
       <StyledDiv>
         We respect your privacy. Please read our{" "}
-        <Link to="/terms-conditions">Terms and Conditions</Link>
+        <Link to="/terms-conditions">Terms and Conditions</Link> and{" "}
+        <Link to="/privacy-policy">Privacy Policy.</Link>
```

Comment:
The reason I opened as a Draft is because I am waiting on the legal team to tell me what's up with this phrase 
`Terms and Conditions and Privacy Policy` <- doesn't make much sense for my broken English (jk jk) I mean it is not grammatically correct in a way so perhaps changing from `Terms and Conditions` to `Terms of Service`

Reply:
Looks like it is fine :) so resolving this!